iaso/tests/api/test_permissions.py

```diff
@@ -0,0 +1,61 @@
+from hat import settings
+from hat.menupermissions.constants import PERMISSIONS_PRESENTATION
+from hat.menupermissions.models import CustomPermissionSupport
+from iaso.test import APITestCase
+from iaso import models as m
+from django.contrib.auth.models import Permission
+
+from iaso.utils.module_permissions import account_module_permissions
+
+
+class OrgUnitAPITestCase(APITestCase):
+    @classmethod
+    def setUpTestData(cls):
+        cls.star_wars = star_wars = m.Account.objects.create(name="Star Wars")
```

Comment:
```suggestion
        cls.account = account = m.Account.objects.create(name="Account")
```

As a best practice, we try to use more meaningful names in unit tests to make it easier to understand.